Plugins/Reqnroll.Verify/Reqnroll.Verify.ReqnrollPlugin/VerifyRuntimePlugin.cs

```diff
@@ -24,12 +28,46 @@ private void RuntimePluginEvents_CustomizeGlobalDependencies(object sender, Cust
     }
 
     private void RuntimePluginTestExecutionLifecycleEvents_BeforeScenario(object sender, RuntimePluginBeforeScenarioEventArgs e)
+    {
+        RegisterGlobal(e);
+
+        RegisterPerScenario(e);
```

Comment:
Instead of calling this from the `LifecycleEvents.BeforeScenario`, we could simply do this registration directly in plugin initialize using the `runtimePluginEvents.CustomizeScenarioDependencies`, like we do here: https://github.com/reqnroll/Reqnroll/blob/main/Plugins/Reqnroll.Autofac.ReqnrollPlugin/AutofacPlugin.cs#L83

With that we would leave the before scenario magic for the legacy behavior only, so the "proper" usage could could be clearer.